x-pack/plugins/watcher/__jest__/client_integration/watch_create_threshold.test.tsx

```diff
@@ -763,17 +763,13 @@ describe('<ThresholdWatchEdit /> create route', () => {
           });
           component.update();
 
-          const latestReqToGetVisualizeData = server.requests.find(
-            (req) => req.method === 'POST' && req.url === '/api/watcher/watch/visualize'
-          );
-          if (!latestReqToGetVisualizeData) {
-            throw new Error(`No request found to fetch visualize data.`);
-          }
-
-          const requestBody = unwrapBodyResponse(latestReqToGetVisualizeData.requestBody);
+          const lastReq: any[] = httpSetup.post.mock.calls.pop() || [];
+          // Options contains two dinamically computed timestamps, so its simpler to just ignore those fields.
+          const { options, ...body } = JSON.parse(lastReq[1].body).watch;
```

Comment:
It took me a minute to understand `lastReq[1]` vs. `lastReq[0]`. It might be clearer to destructure:

```
const [requestUrl, watchBody] = lastReq;
```